python/sparkdl/graph/graph_builder.py

```diff
@@ -0,0 +1,200 @@
+#
```

Comment:
could we name this file builder.py and if needed do 
```
import sparkdl.graph.builder as graph_builder
```
?

is there a convention @MrBago  ?